ui/app/components/transit-key-actions.js

```diff
@@ -3,15 +3,30 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import { assign } from '@ember/polyfills';
-import { copy } from 'ember-copy';
-import { assert } from '@ember/debug';
+import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
-import Component from '@ember/component';
-import { set, get, computed } from '@ember/object';
+import { action } from '@ember/object';
+import { assert } from '@ember/debug';
+import { tracked } from '@glimmer/tracking';
+import { task } from 'ember-concurrency';
+import { waitFor } from '@ember/test-waiters';
 import { encodeString } from 'vault/utils/b64';
 
-const TRANSIT_PARAMS = {
+/**
+ * @module TransitKeyActions
+ * TransitKeyActions component handles the actions a user can take on a transit key model. The model and props are updated on every tab change
+ *
+ * @example
+ * ```js
+ * <TransitKeyActions
+ * @key={{this.model}}
+ * @selectedAction="hmac"
+ * />
```

Comment:
mostly an FYI the proper formatting for jsdoc `@example` is no code snippet back tick marks (so this can be removed: ` ```js `). You can take a look at my docfy hackweek PR [here](https://github.com/hashicorp/vault/pull/24092/files) to see how I had to modify jsdoc  
```suggestion
 * @example
 * <TransitKeyActions
 * @key={{this.model}}
 * @selectedAction="hmac"
 * />
```

Reply:
TIL, thank you.